Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -12352,14 +12343,24 @@ - (void)_insertDynamicImageAnalysisContextMenuItemsIfPossible
             }
         }
 
-        if (self.hasSelectableTextForImageContextMenu)
-            addAction(_WKElementActionTypeImageExtraction);
+        if (!foundShowTextItem && self.hasSelectableTextForImageContextMenu) {
+            // Dynamically insert the "Show Text" menu item, if it wasn't already inserted.
+            // FIXME: This should probably be inserted unconditionally, and enabled if needed
```

Comment:
Sorry if this is a stupid question, but why not just doit this way now? Too complicated?